### PR TITLE
Implement batchNormalization.

### PR DESCRIPTION
**Pull request by BruceDai**
@huningxin PTAL, thanks.

### CODE REVIEW

File: src/batch_normalization.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import {reshape} from './reshape.js';
+import {Tensor} from './tensor.js';
+
+/**
+ * Validate the tensor whether it is a 1-D tensor and its length is equal to `expectedSize`.
+ * @param {Tensor} a
+ * @param {Number} expectedSize
+ * @param {String} name
+ */
+function validateTensor(a, expectedSize, name) {
```
Comment (huningxin) [inline src/batch_normalization.js:13]: I prefer to a more specific name, such as `check1DTesnorWithSize`.

**Review by huningxin — APPROVED** [review]
LGTM with one nit. PTAL.

File: src/batch_normalization.js (hunk 1 above)
Comment (BruceDai) [reply src/batch_normalization.js]: Thanks @huningxin. 
Updated it. Please take a look again, thanks.

Comment (huningxin) [reply src/batch_normalization.js]: LGTM
